db/migrate/20210914103636_change_user_orders_devices_default.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeUserOrdersDevicesDefault < ActiveRecord::Migration[6.1]
+  def change
+    change_column_default :users, :orders_devices, from: nil, to: false
```

Comment:
Not sure whether to keep this migration reversible like this. Developer thoughts?